sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelperTest.java

```diff
@@ -363,4 +378,122 @@ public void truncateCauses() {
         }
         assertEquals(ErrorLogHelper.CAUSE_LIMIT, depth);
     }
+
+    @Test
+    public void getStoredDeviceInfo() {
+        String deviceInfoString = "{\"sdkName\":\"appcenter.android\",\"sdkVersion\":\"2.5.4.2\",\"model\":\"Android SDK built for x86\",\"oemName\":\"Google\",\"osName\":\"Android\",\"osVersion\":\"9\",\"osBuild\":\"PSR1.180720.075\",\"osApiLevel\":28,\"locale\":\"en_US\",\"timeZoneOffset\":240,\"screenSize\":\"1080x1794\",\"appVersion\":\"2.5.4.2\",\"carrierName\":\"Android\",\"carrierCountry\":\"us\",\"appBuild\":\"59\",\"appNamespace\":\"com.microsoft.appcenter.sasquatch.project\"}";
+        File deviceInfoFile = mock(File.class);
+        File minidumpFolder = mock(File.class);
+        when(minidumpFolder.listFiles(any(FilenameFilter.class))).thenReturn(new File[]{deviceInfoFile});
+        mockStatic(FileManager.class);
+        when(FileManager.read(eq(deviceInfoFile))).thenReturn(deviceInfoString);
+        Device storedDeviceInfo = ErrorLogHelper.getStoredDeviceInfo(minidumpFolder);
+        assertNotNull(storedDeviceInfo);
+    }
+
+    @Test
+    public void getStoredDeviceInfoNull() {
+
+        /* Test null directory. */
+        File minidumpFolder = mock(File.class);
```

Comment:
Same

Reply:
I can't achieve 100% coverage without it for the moment.